.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/kframework/k
 	ignore = untracked
 [submodule "tests/wasm-tests"]
-	path = tests/wasm-tests
+	path = deps/spec
```

Comment:
Either (1) don't do this, or (2) also change the submodule name (line above this).

I would lean towards (1), we're only using the spec repo as tests, not as dependencies.